javatests/jsinterop/generator/externs/simpleclass/simpleclass.js

```diff
@@ -69,11 +69,24 @@ SimpleClass.wait = function() {};
  */
 SimpleClass.staticReadonlyProperty
 
+/**
+ * @deprecated
+ * @const {string}
+ */
+SimpleClass.staticDeprecatedProperty
```

Comment:
Don't forget the `;` at the end of the line

Reply:
done